mlir/lib/Dialect/Rock/Transforms/ConvToGemm.cpp

```diff
@@ -474,10 +499,14 @@ struct MatchLayoutsToInput final
     TypedValue<ShapedType> filter = op.getFilter(), output = op.getOutput();
     const llvm::StringMap<StringAttr> inputToFilter = {
         {"ci", b.getStringAttr("c")},
+        {"0i", b.getStringAttr("0")},
```

Comment:
Need both old and new forms.  Without this change, resnet18_blk.mlir has an RMS of 1e-3 instead of 1e-7.